src/pages/library/data-catalog/CreateDataset/CreateDatasetInputs.test.ts

```diff
@@ -101,6 +110,12 @@ describe('CreateDatasetInputs', () => {
     // Assert
     expect(screen.getByText('Title')).toBeTruthy();
     // Cannot test for value because the it gets into testing underlying library behavior
```

Comment:
Seems like this comment is out-of-date now.

Reply:
Removed in https://github.com/DataBiosphere/terra-ui/pull/4020/commits/4de1a3205cca3a70e7bdfe2b55d052703f5873a9.